bin/blockfrostUtils.js

```diff
@@ -0,0 +1,58 @@
+import { readFile } from 'fs/promises';
+import { BlockFrostAPI } from '@blockfrost/blockfrost-js';
+
+/**
+ *
+ * @param {string} blockfrostKey - blockfrostKey for the transaction.
+ * @param {string} walletAddress - wallet address for the transaction.
+ * @returns {string, number, number} txHash, txIx, amount of the transaction.
+ */
+const queryUTxOviaBlockfrost = async (blockfrostKey, walletAddress) => {
+  const blockfrost = new BlockFrostAPI({
+    projectId: blockfrostKey,
+  });
+  const utxos = await blockfrost.addressesUtxosAll(walletAddress);
+  const {
+    tx_hash: txHash = '',
+    tx_index: txIx = 0,
+    sum,
+  } = utxos
+    .map((utxo) => ({
+      ...utxo,
+      sum: utxo?.amount.reduce((acc, item) => acc + parseInt(item?.quantity, 10), 0),
+    }))
+    .find((utxo) => utxo.sum > 1000000);
```

Comment:
I made a small change to the logic here: `sum` gets calculated only once, previously it was done twice. Guys please confirm you agree with this change.

Reply:
Looks good to me.